nitime/tests/test_algorithms.py

```diff
@@ -145,11 +145,11 @@ def test_psd_matlab():
     ts = np.loadtxt(os.path.join(test_dir_path, 'tseries12.txt'))
 
     #Complex signal!
-    ts0 = ts[1] + ts[0] * np.complex(0, 1)
+    ts0 = ts[1] + ts[0] * complex(0, 1)
 
     NFFT = 256
     Fs = 1.0
-    noverlap = NFFT / 2
+    noverlap = NFFT // 2
```

Comment:
Not sure when float `noverlap` stopped being okay, but given this is always 128, this seems fine.